src/main/java/browserview/ChromeDriverEx.java

```diff
@@ -50,9 +52,7 @@ public void get(String url, boolean isForceRefresh) throws WebDriverException {
             logger.info("Test loading page: " + url);
             testGet();
         } else {
-            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url) ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/commits") ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/files"))) {
+            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url))) {
```

Comment:
We don't have a test that covers all the browser action. Changing this here could cause a lot of trouble, unless you have exhaustively test all browser action. You need to change your implementation so that this can stay as it is.